lib/kramdown_rpf/rpf.rb

```diff
@@ -110,6 +112,31 @@ def self.convert_print_only_to_html(content)
         ::Kramdown::Document.new("<div class=\"u-print-only\">\n#{content}</div>", KRAMDOWN_OPTIONS).to_html
       end
 
+      def self.convert_quiz_to_html(quiz)
+        quiz =~ YAML_FRONT_MATTER_REGEXP
+        details = YAML.safe_load(Regexp.last_match(1))
+        question = details['question']
+        choices = YAML.safe_load(Regexp.last_match(2))
+        radios = choices.map { |c| c =~ RADIO_REGEXP ? $1 : c }.reject(&:empty?)
+        radio_inputs = radios.map.with_index(1) { |c, i| "<label class=\"c-project-quiz__label\" for=\"choice-#{i}\">#{c}</label>\n      <input class=\"c-project-quiz__input\" name=\"quiz-choice\" type=\"radio\" id=\"choice-#{i}\" value=\"choice-#{i}\" />\n      " }.join("").strip
```

Comment:
This line is rather.. lengthy.  Maybe the `.join.strip` could go where in the heredoc?

TBH, I'm wondering if a heredoc would make sense for this loop too, except your careful whitespace would get mangled..